x-pack/plugins/osquery/public/live_queries/form/index.tsx

```diff
@@ -53,14 +56,20 @@ const LiveQueryFormComponent: React.FC<LiveQueryFormProps> = ({
         body: JSON.stringify(payload),
       }),
     {
-      onSuccess,
+      onSuccess: () => {
+        setErrorToast();
+        if (onSuccess) {
+          onSuccess();
+        }
+      },
       onError: (error) => {
-        // @ts-expect-error update types
-        toasts.addError(error, { title: error.body.error, toastMessage: error.body.message });
+        setErrorToast(error);
       },
     }
   );
 
+  const expirationDate = useMemo(() => new Date(data?.actions[0].expiration), [data?.actions]);
```

Comment:
can we make sure we're working on ISO dates? 
my usecase:
my browser for `new Date()` will return a date in my local time vs date on ES cluster can be different, so I think it's going to be safer if we do compare ISO strings

Reply:
i think this situation will be fine. the dates being grabbed from ES are ISO and converted into a js date, which coerces the format into UTC under the hood. since the date constructed from the ES document and the new date both in UTC the comparison should work as expected. i compared the date that gets constructed to make sure it is 1 day after the time the action was created (casted to the proper timezone), and it seems to work as expected.

it seems like the care should be taken when creating the record (though i think the use of `now` and an offset would avoid any problems?)